### ISSUE TITLE
Aergo Connect displays 0 balance for my Ledger account

### ISSUE THREAD

**Issue by Netucet** [aergoio/aergo-connect#43]

**Comment by kroggen** [comment]
I suspect that you can safely delete the second account with the same address (bellow) and leave only the first one, with the USB icon

**Comment by Netucet** [comment]
Yes, I am not worried about deleting the second address. The question is - why this happened? Why the same address is once recognized and shortly after isn't. Might be a bug in the extension, so just wanted to let you know about this.
